pkgs/data/fonts/noto-fonts/default.nix

```diff
@@ -61,6 +60,44 @@ let
         maintainers = with maintainers; [ mathnerd314 emily ];
       };
     };
+
+  mkNotoCJK = { typeface, version }:
+    stdenvNoCC.mkDerivation {
+      pname = "noto-fonts-cjk-${lib.toLower typeface}";
+      inherit version;
+
+      src = fetchFromGitHub {
+        owner = "googlefonts";
+        repo = "noto-cjk";
+        rev = "9f7f3c38eab63e1d1fddd8d50937fe4f1eacdb1d";
```

Comment:
I think this should be a parameter as well. Otherwise people may change the version parameter and be confused if the result remains the same.

Reply:
I made `rev` and `sha256` parameters to `mkNotoCJK`.